eth/tracers/tracer.go

```diff
@@ -578,16 +579,16 @@ func (jst *Tracer) CaptureEnd(output []byte, gasUsed uint64, t time.Duration, er
 
 //@NOTE:SHYFT
 type Internals struct {
-	Type string
-	From string
-	To string
-	Value string
-	Gas string
+	Type 	string
+	From 	string
+	To 		string
```

Comment:
sapcing

Reply:
Weird, when I open my editor I do not have that spacing issue.